bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/services/dto/BoschSHCServiceState.java

```diff
@@ -37,7 +37,7 @@ public class BoschSHCServiceState {
     @SerializedName("@type")
     public final String type;
 
-    protected BoschSHCServiceState(String type) {
+    public BoschSHCServiceState(String type) {
```

Comment:
Minor, but i don't see yet why protected isn't enough?

Reply:
I used the constructor in the BoschServiceDataDeserializer class for the default return value (if LongPollResult does not consist of DeviceServiceData or Scenario)